homeassistant/components/almond/strings.json

```diff
@@ -1,15 +1,17 @@
 {
   "config": {
     "step": {
-      "pick_implementation": { "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]" },
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
+      },
       "hassio_confirm": {
         "title": "Almond via Hass.io add-on",
         "description": "Do you want to configure Home Assistant to connect to Almond provided by the Hass.io add-on: {addon}?"
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Almond account.",
-      "cannot_connect": "Unable to connect to the Almond server.",
+      "already_setup": "[%key:common::config_flow::abort::already_configured_account%]",
```

Comment:
You need to change this to `single_instance_allowed` and also use the right reference string.

Reply:
Goot catch. Had unpushed commit. Pushed now.